organization/src/main/scala/com/improving/app/organization/domain/Organization.scala

```diff
@@ -1,43 +1,6 @@
 package com.improving.app.organization.domain
 
-import com.improving.app.organization.api
-import kalix.scalasdk.eventsourcedentity.EventSourcedEntity
-import kalix.scalasdk.eventsourcedentity.EventSourcedEntityContext
+object Organization{
```

Comment:
This object is now empty. What's its purpose?

Reply:
it contained the OrganizationId - its not needed any more